antora-playbook.yml

```diff
@@ -10,6 +10,7 @@ site:
     docsearch_id: 'QK2EAH8GB0'
     docsearch_api: 'ef7bd9485eafbd75d6e8425949eda1f5'
     docsearch_index: 'prod_hazelcast_docs'
+    # KapaAI API_KEY
```

Comment:
?

Reply:
It was a "technical debt". Some time ago I was describing all setting pieces of this file and forgot to add a note about KapaAI :)